meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/ti-sci-fw/ti-sci-fw_git.bb

```diff
@@ -0,0 +1,121 @@
+require recipes-bsp/ti-linux-fw/ti-linux-fw.inc
```

Comment:
We don't need to duplicate this recipe, all we need is to define TI_K3_SECDEV_INSTALL_DIR, which can be done via a bbappend (assuming we will be using this with latest meta-ti kirkstone).

Reply:
right, I will move it to a bbappend